docs/G-Codes.md

```diff
@@ -160,8 +160,9 @@ The following standard commands are supported:
   Wait until the given temperature sensor is at or above the supplied
   MINIMUM and/or at or below the supplied MAXIMUM.
 - `SET_VELOCITY_LIMIT [VELOCITY=<value>] [ACCEL=<value>]
-  [ACCEL_TO_DECEL=<value>] [SQUARE_CORNER_VELOCITY=<value>]`: Modify
-  the printer's velocity limits.
+  [ACCEL_TO_DECEL=<value>] [SQUARE_CORNER_VELOCITY=<value>]
+  [RESPOND=0]`: Modify the printer's velocity limits. If RESPOND=0 is
```

Comment:
This new param is no longer supported.

Reply:
Oh... Yes! I only changed the command for testing.